src/main/java/jenkins/scm/api/SCMNavigatorDescriptor.java

```diff
@@ -41,6 +42,40 @@ protected SCMNavigatorDescriptor(Class<? extends SCMNavigator> clazz) {
     }
 
     /**
+     * A description of this kind of {@link SCMNavigator}. This description can contain HTML code but it is recommend
+     * to use text plain in order to avoid how it should be represented.
+     *
+     * @return A string, by default the value from newInstanceDetail view is taken.
+     *
+     * @since TODO
+     */
+    @Nonnull
+    public abstract String getDescription();
```

Comment:
@jglick After this [commit](https://github.com/jenkinsci/scm-api-plugin/pull/9/commits/5b289d622c0619298c33d0d5c8bba0ec338a7b08) I have to work in this method.